x/leverage/keeper/iter.go

```diff
@@ -0,0 +1,228 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/umee-network/umee/x/leverage/types"
+)
+
```

Comment:
I have refactored all iterators in `x/leverage` to use the `k.iterate` helper which @RafilxTenfen made.

They have been moved to this file, with the exception of the ones that export genesis-related structs (those are in genesis.go).